splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/operations/export/ExportOperation.java

```diff
@@ -208,7 +208,15 @@ public void writeExternal(ObjectOutput out) throws IOException {
             case "parquet": {
                 OperationContext<?> writeContext = dsp.createOperationContext(this);
                 long start = System.currentTimeMillis();
-                dataset.writeParquetFile(dsp, new int[]{}, exportParams.getDirectory(), exportParams.isCompression() ? "snappy" : "none", writeContext);
+                String compression = null;
+                if (exportParams.getCompression() == ExportFile.COMPRESSION.SNAPPY) {
```

Comment:
We should handle gz and bzip2 here too

Reply:
Parquet does not support bzip2 compression